### PR TITLE
SDAP-511 - Switch package manager to poetry

### PR DESCRIPTION
**Pull request by RKuttruff**

### CODE REVIEW

**Review by skorper — COMMENTED** [review]
Shouldn't we also change collection manager?

**Comment by RKuttruff** [comment]
> Shouldn't we also change collection manager?

We should, but I left it as-is for now, since it wasn't causing any issues re: GPL code

File: granule_ingester/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+scipy = "1.7.3"
+netcdf4 = "1.5.3"
+pandas = "1.3.5"
+pytz = "2019.3"
+xarray = "*"
+pyyaml = "5.3.1"
+aiohttp = ">=3.8.0"
+tenacity = "8.2.3"
+requests = ">=2.27.1"
+
```
Comment (skorper) [inline granule_ingester/pyproject.toml:56]: No dev dependencies?

**Comment by skorper** [comment]
Same as nexus, I think we should remove VERSION.txt

**Comment by skorper** [comment]
> > Shouldn't we also change collection manager?
> 
> We should, but I left it as-is for now, since it wasn't causing any issues re: GPL code

No time like the present! If it's not too much work (I know you've already done a ton of work on this) I think we should change it over. I fear we'd never fix it if we don't just do it now

File: granule_ingester/pyproject.toml (hunk 1 above)
Comment (RKuttruff) [reply granule_ingester/pyproject.toml:56]: I don't think GI uses anything additional for testing, etc

**Comment by RKuttruff** [comment]
> > > Shouldn't we also change collection manager?
> > 
> > 
> > We should, but I left it as-is for now, since it wasn't causing any issues re: GPL code
> 
> No time like the present! If it's not too much work (I know you've already done a ton of work on this) I think we should change it over. I fear we'd never fix it if we don't just do it now

Probably won't be _too_ much work, though more than I can do today.

**Comment by RKuttruff** [comment]
> Same as nexus, I think we should remove VERSION.txt

This would be dependent on whether CM switches over to using poetry too

**Comment by skorper** [comment]
> > > > Shouldn't we also change collection manager?
> > > 
> > > 
> > > We should, but I left it as-is for now, since it wasn't causing any issues re: GPL code
> > 
> > 
> > No time like the present! If it's not too much work (I know you've already done a ton of work on this) I think we should change it over. I fear we'd never fix it if we don't just do it now
> 
> Probably won't be _too_ much work, though more than I can do today.

Okay, let's push this to the next release. This is already a _huge_ improvement

**Review by skorper — APPROVED** [review]

**Comment by RKuttruff** [comment]
Merged. Thanks for your help with this transition!

We should be all caught up to get RC4 started.

**Pull request merged**